context.jsonld

```diff
@@ -57,6 +57,18 @@
         "maxVelocityAxis4": "https://smartdatamodels.org/dataModel.RoboticIndustrialActivities/maxVelocityAxis4",
         "maxVelocityAxis5": "https://smartdatamodels.org/dataModel.RoboticIndustrialActivities/maxVelocityAxis5",
         "maxVelocityAxis6": "https://smartdatamodels.org/dataModel.RoboticIndustrialActivities/maxVelocityAxis6",
+        "measuredCurrentAxis1": "https://smartdatamodels.org/dataModel.RoboticIndustrialActivities/measuredCurrentAxis1",
```

Comment:
you do not need to edit this file, this is created automatically, once approved the schema